### PR TITLE
S4 create page

### PR DESCRIPTION
**Pull request by ucanfil**
Everything needs to be reviewed inside the files:
index.php
index.scss

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -25,32 +24,29 @@
...
-                                    </div>
-                                    <div class="icon-container">
-                                        <img src="img/Eye_opened.svg" class="icon">
-                                        <p>0</p>
-                                    </div>
-                                    <div class="icon-container">
-                                        <img src="img/pokeball-2.svg" class="icon">
-                                        <p>0</p>
-                                    </div>
+                            <div class="poke-attributes">
```
Comment (Benson-J) [inline index.php]: There should be no unseen/seen/caught content here, that doesn't get added until a later story.

File: scss/index.scss [hunk 2]
```diff
@@ -72,17 +76,17 @@ h2 {
 }
 
 .poke-attributes {
```
Comment (Benson-J) [inline scss/index.scss]: Same as above, should not be present in this story.

File: scss/index.scss [hunk 3]
```diff
@@ -72,17 +76,17 @@ h2 {
...
-  overflow: auto;
+  margin: auto;
+  width: 50%;
+  height: 100px;
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-end;
 }
 
 .icon-container {
```
Comment (Benson-J) [inline scss/index.scss]: Same as above, should not be present in this story.

File: scss/index.scss [hunk 4]
```diff
@@ -72,17 +76,17 @@ h2 {
...
   float: left;
   height: 40%;
-  padding: 3% 12% 0 12%;
 }
 
 .icon {
```
Comment (Benson-J) [inline scss/index.scss]: Same as above, should not be present in this story.

**Review by Benson-J — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (ucanfil) [reply index.php]: Solved in > 881776f

File: scss/index.scss (hunk 2 above)
Comment (ucanfil) [reply scss/index.scss]: Solved in > 881776f

File: scss/index.scss (hunk 3 above)
Comment (ucanfil) [reply scss/index.scss]: Solved in > 881776f

File: scss/index.scss (hunk 4 above)
Comment (ucanfil) [reply scss/index.scss]: Solved in > 88fa222

**Review by Benson-J — APPROVED** [review]

**Pull request merged**